src/measurements/CalibrationClasses.py

```diff
@@ -311,11 +312,12 @@ class ChirpCalibrationMeasurement(QtCore.QThread):
             - sendProgress: float representing the progress of the measurement.
     '''
     sendSpectrum = QtCore.pyqtSignal(np.ndarray, np.ndarray)
-    send_chirp= QtCore.pyqtSignal(np.ndarray,np.ndarray,np.ndarray)
+    send_chirp = QtCore.pyqtSignal(np.ndarray,np.ndarray,np.ndarray)
     send_chirp_calibration_data = QtCore.pyqtSignal(tuple)
+    send_beam = QtCore.pyqtSignal(object)
     sendProgress = QtCore.pyqtSignal(float)
 
-    def __init__(self,devices, background, grating_period, beam_, compression_carrier_wavelength, chirp_step, chirp_max, chirp_min,beam, demo=False):
+    def __init__(self,devices, background, grating_period, compression_carrier_wavelength, chirp_step, chirp_max, chirp_min, beam_name, beam, spectral_calibration=None, demo=False):
         '''
          Initializes the semporal beam calibration measurement
```

Comment:
Not all inputs are described in the docstring. Please add the missing qualifiers